### ISSUE TITLE
Toggle

### ISSUE THREAD

**Issue by Libertes** [wet-boew/wet-boew#8208]
There is bug with the toggle in IE11, when the URL contains the "%" character, the toggle doesn't work anymore.

See here: 
http://wet-boew.github.io/wet-boew/demos/toggle/toggle-fr.html?%

**Comment by duboisp** [comment]
You have a malformed URI.
The ```%``` need to be encoded as ```%25```

Example: http://wet-boew.github.io/wet-boew/demos/toggle/toggle-fr.html?%25

Reference: https://tools.ietf.org/html/rfc3986#section-2.4

**Comment by Libertes** [comment]
Yes you are right, my example wasn't good.  What if I encode the word "déterminé" ?

It will give me this : d%E9termin%E9, which will break the toggle
http://wet-boew.github.io/wet-boew/demos/toggle/toggle-fr.html?d%E9termin%E9

**Comment by duboisp** [comment]
Again, you have a malformed URI.

"déterminé" when encoded is "d%C3%A9termin%C3%A9"

So the URL would be
http://wet-boew.github.io/wet-boew/demos/toggle/toggle-fr.html?d%C3%A9termin%C3%A9

Which do not break the toggle plugin.

Tip: The malformed URI error is displayed in the browser error console. ```Uncaught URIError: URI malformed```
